src/components/CmsZone.vue

```diff
@@ -196,6 +195,17 @@ export default class CmsZone extends Vue {
     this.$el.classList.remove('cms-zone-error');
     this.$el.classList.remove('cms-zone-offline');
 
+    this.removeScrollListeners();
+    this.scrollable = getClosest(this.$el, '.scrollable-content') || document.body;
+    const unbind = this.addScrollListener((): void => {
+      if (this.$el && !this.nonce && this.isAlmostVisible()) {
```

Comment:
are we using `nonce` to see if we've fetched the zone before? if so, i think i'd prefer a slightly more explicit variable to track this

Reply:
How about a getter that returns true if the nonce is 0? Any name suggestions?